example/javascriptlib/basic/5-client-server-hello/build.mill

```diff
@@ -0,0 +1,61 @@
+package build
+import mill._, javascriptlib._
+
+object client extends JestModule
+
+object server extends JestModule
+
+object app extends TypeScriptModule {
+  def moduleDeps = Seq(client, server)
+
+  def bundleApp =
+    Task {
+      val serverPath = server.bundle().path
+      val clientPath = client.bundle().path
+      println(s"client bundled@$clientPath\nserver bundled@$serverPath")
+      (PathRef(serverPath), PathRef(clientPath))
+    }
+
+}
+
+// Documentation for mill.example.javascriptlib
+
+/** Usage
+
+> mill client.test
+PASS .../client.test.ts
+...getServerResponse
+...should return mocked response...
+Test Suites:...1 passed, 1 total...
+Tests:...1 passed, 1 total...
+Snapshots:...
+Time:...
+Ran all test suites matching ...
+
+> mill server.test
+PASS .../server.test.ts
+...Server Tests
+...GET /hello should return "Hello from the server!"...
+...
+Test Suites:...
+Tests:...
+Snapshots:...
+Time:...
+Ran all test suites matching ...
+
+> mill show app.bundleApp
+...
+...
+...
+...
+client bundled@...client/bundle.dest/bundle.js
+server bundled@...server/bundle.dest/bundle.js
+...
+
+> node out/server/bundle.dest/bundle.js & echo "hello" | node out/client/bundle.dest/bundle.js &  sleep 2 && kill -9 $(lsof -t -i:3000) # 1) start server 2) start client 3) send client input 4) kill server
```

Comment:
- Let's move the `#` comment onto a separate line before so it's more readable
- Is there a `server.run` command or something we can use rather than calling `node` on the binaries directly? That would make the example a lot easier to skim

Reply:
yes there is a server.run, will modify accordingly

Reply:
I can't run two mill operations at the same time 
so this line:

`mill server.run & echo "hello" | mill client.run & sleep 2 && kill -9 $(lsof -t -i:3000)`

will not work as intended,
plus `echo "hello" | mill client.run` will probably not pass the "hello" argument to the client.

Also, trying to seprate the node commands will cause the test to stall as the command needs to exit before the next and seeing as it is a server, it wont exit


So do I just remove that section from the documentation?